astropy/uncertainty/distributions.py

```diff
@@ -44,8 +44,10 @@ def normal(center, *, std=None, var=None, ivar=None, n_samples,
 
     Returns
     -------
-    distr : ``cls``, usually `Distribution`
+    distr : `Distribution` or object
         The sampled Gaussian distribution.
+        Type specified by ``cls``.
```

Comment:
'The type will be the same as the current class' or something like this?

Reply:
✅